.buildkite/x-pack/pipeline.xpack.packetbeat.yml

```diff
@@ -26,46 +25,262 @@ env:
-
-  - label: ":linux: Load dynamic x-pack packetbeat pipeline"
-    key: "packetbeat-pipeline"
-    command: ".buildkite/scripts/generate_xpack_packetbeat_pipeline.sh"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-beats-ci:latest"
-    notify:
-      - github_commit_status:
-          context: "${BEATS_PROJECT_NAME}: Load dynamic pipeline's steps"
+      - label: ":linux: Ubuntu System Tests"
+        key: "mandatory-linux-system-test"
+        command: |
+          cd x-pack/packetbeat
+          mage systemTest
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Ubuntu System Tests"
+
+      - label: ":rhel: RHEL9 Unit Tests"
+        key: "mandatory-rhel9-unit-test"
+        command: |
+          cd x-pack/packetbeat
+          mage build unitTest
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_RHEL9_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: RHEL9 Unit Tests"
+
+      - label: ":windows: Windows 2016 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/packetbeat
+          mage build unitTest
+        key: "mandatory-win-2016-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2016}"
+          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Windows 2016 Unit Tests"
+
+      - label: ":windows: Windows 2022 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/packetbeat
+          mage build unitTest
+        key: "mandatory-win-2022-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2022}"
+          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Windows 2022 Unit Tests"
+
+      #   #skip: "see https://github.com/elastic/beats/issues/38142"
+      # - label: ":windows: Windows 2022 System Tests"
+      #   key: "mandatory-win-2022-system-tests"
+      #   command: ".buildkite/scripts/win_unit_tests.ps1 systemtest"
```

Comment:
This command was removed in https://github.com/elastic/beats/pull/38583/files#diff-4162b08475e2b8bdb77ca99545f27d4b5c0a32c38d2a2ac22e5aea16fe60900d

maybe we can leave a comment with ^^ link so that we can find what it needs to run (essentially, set up google_cloud_auth and run `mage systemTest`)

Reply:
changed